install/docker-extension/ui/src/components/Catalog/CatalogCard.js

```diff
@@ -25,9 +25,10 @@ function CatalogCard({ pattern, patternType, catalog }) {
   const handleCopyClick = async event => {
     event.preventDefault();
 
+    let name = pattern?.name?.toLowerCase().split(' ').join('-')
     try {
       await navigator.clipboard.writeText(
-        `${mesheryCloudUrl}/catalog/${pattern.id}`
+        `${mesheryCloudUrl}/catalog/content/${name}-${pattern.id}?source=%257B%257D`
```

Comment:
When this name changes, the link will break.

@Yashsharma1911, @captain-Akshay 


@humblenginr use this format instead - http://meshery.layer5.io/catalog/content/design/3a0dfed8-e5ff-4d47-94c4-af4e22c9704a

Reply:
No we don't need to add name with id, @humblenginr you can remove this instead have 
```
`${mesheryCloudUrl}/catalog/content/${pattern.id}
```